lucene/grouping/src/java/org/apache/lucene/search/grouping/TopGroupsCollector.java

```diff
@@ -39,6 +39,14 @@
   private final Sort withinGroupSort;
   private final int maxDocsPerGroup;
 
+  protected TopGroupsCollector(GroupReducer<T, TopDocsCollector<?>> groupReducer, GroupSelector<T> groupSelector, Collection<SearchGroup<T>> groups, Sort groupSort, Sort withinGroupSort,
+                            int maxDocsPerGroup, boolean getScores, boolean getMaxScores, boolean fillSortFields) {
+    super(groupSelector, groups, groupReducer);
```

Comment:
You can remove a few of these parameters now, I think?  They're for the Reducer, and as such don't need to be passed separately.